include/extractor/original_edge_data.hpp

```diff
@@ -46,7 +46,7 @@ struct OriginalEdgeData
     }
 
     OriginalEdgeData()
-        : via_geometry{std::numeric_limits<unsigned>::max(),false},
+        : via_geometry{std::numeric_limits<unsigned>::max() >> 1,false},
```

Comment:
also these additional locations are easy to mess up if the ID values should change